strictdoc/server/routers/main_router.py

```diff
@@ -2696,6 +2698,49 @@ def get_search(q: Optional[str] = None):
             status_code=200,
         )
 
+    # @relation(SDOC-SRS-120, scope=function)
+    @router.get("/autocomplete/uid", response_class=Response)
+    def get_autocomplete_results(q: Optional[str] = None):
+        output = ""
+        if q is not None:
+            query_words = q.lower().split()
+            resulting_nodes = []
+            for (
+                document
+            ) in export_action.traceability_index.document_tree.document_list:
+                document_iterator = (
+                    export_action.traceability_index.get_document_iterator(
+                        document
+                    )
+                )
+                for node in document_iterator.all_content(
+                    print_fragments=False, print_fragments_from_files=False
+                ):
+                    if node.reserved_uid is not None:
```

Comment:
Here you also need to exclude the possibility that a node is a section because otherwise I can try to link a node to a section which is currently not allowed and an assert is hit.

The easiest is to just:

```py
if node_.is_section:
    continue
```